index.js

```diff
@@ -43,7 +43,8 @@ function DB (opts) {
       if (v && v.lat !== undefined && v.lon !== undefined) {
         next(null, { type: 'put', point: ptf(v) })
       } else if (d && Array.isArray(row.value.points)) {
-        next(null, { type: 'del', points: row.value.points.map(ptf) })
+        var pt = row.value.points.map(ptf)[0]
```

Comment:
We need an index record for each point in `row.value.points`. This property is set in the delete code and is an array of the lat/lon of each parent. Each of those records should be updated to point to the deleted record, i.e. we need two `put`s